Mathlib/Testing/SlimCheck/Testable.lean

```diff
@@ -509,8 +509,8 @@ open Lean
 quantifiers and add `NamedBinder` annotations next to them. -/
 partial def addDecorations (e : Expr) : MetaM Expr :=
   Meta.transform e fun expr => do
-    if not (← Meta.inferType e).isProp then
-      return .continue
+    if not (← Meta.inferType expr).isProp then
+      return .done expr
```

Comment:
I find the meaning clearer this way (although I know you weren't the one introducing this spelling.
```suggestion
    unless (← Meta.inferType expr).isProp
      return .done expr
```

Reply:
Does `unless` support `else`? I think in an `else`/`if` chain, using `unless` makes things a little weirder to read.

Reply:
I don't think it does, but I claim that's a feature. The `unless` clause indicates a condition on which you want to immediately abort, which is not what an `else`/`if` chain indicates.

Reply:
I'm going to use "this if isn't really part of this patch" to avoid having to do anything in this CL. If you feel strongly that `unless` is underused, perhaps open a follow-up with 10 or so replacements across mathlib and see what people think.

bors merge